package.json

```diff
@@ -9,7 +9,7 @@
     },
     "scripts": {
         "start": "tsc && node dist/app.js",
-        "test": "tsc && jest",
+        "test": "jest",
```

Comment:
tsc not needed anymore?

Reply:
It is needed, thats the reason I included it here in an earlier PR. So this removal was an error, thanks for spotting it.